.circleci/config.yml

```diff
@@ -99,23 +103,27 @@ workflows:
   build-test:
     jobs:
       - build-test:
-          <<: *python-version-matrix
+          name: build-py<< matrix.python-version >>-Django<< matrix.django-version >>
```

Comment:
Maybe call this `build-test` for clarity? (also the other one with the same name below)

Reply:
c8abf38